common/sgx/tcbinfo.c

```diff
@@ -503,7 +503,10 @@ oe_result_t oe_parse_tcb_info_json(
     if (itr == end)
     {
         if (platform_tcb_level->status != OE_TCB_LEVEL_STATUS_UP_TO_DATE)
-            OE_RAISE(OE_TCB_LEVEL_INVALID);
+            OE_RAISE_MSG(
+                OE_TCB_LEVEL_INVALID,
+                "Platform TCB (%d) is not up-to-date",
+                platform_tcb_level->status);
```

Comment:
This is nice. Will be useful for our tests in the coming days.

Reply:
Do you think it might be worthwhile to print the entire oe_tcb_level_t struct on failure here?  It has these fields:
```
typedef struct _oe_tcb_level
{
    uint8_t sgx_tcb_comp_svn[16];
    uint16_t pce_svn;
    oe_tcb_level_status_t status;
} oe_tcb_level_t;
```